### PR TITLE
Angular 16 support

### PR DESCRIPTION
**Pull request by GuillaumeNury**
## Description

Bump all dependencies.

-----

### CODE REVIEW

File: jest.config.json [hunk 1]
```diff
@@ -1,7 +1,6 @@
 {
 	"preset": "jest-preset-angular",
 	"setupFilesAfterEnv": ["<rootDir>/setup-jest.ts"],
-	"globalSetup": "jest-preset-angular/global-setup",
```
Comment (shajz) [inline jest.config.json:4]: not needed anymore ? 🎉

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-2114/storybook

**Review by shajz — APPROVED** [review]

File: .eslintrc [hunk 2]
```diff
@@ -54,7 +54,7 @@
 					}
 				],
 				"curly": "error",
-				"@delagen/deprecation/deprecation": "warn",
+				"deprecation": "warn",
```
Comment (shajz) [inline .eslintrc:57]:
```suggestion
				"deprecation/depreciation": "warn",
```

see https://github.com/gund/eslint-plugin-deprecation#setup

File: jest.config.json (hunk 1 above)
Comment (GuillaumeNury) [reply jest.config.json:4]: nope, after NG16, there is no more `ngcc` compilation step

**Review by shajz — DISMISSED** [review]

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-2114/storybook

**Review by jeremie-lucca — APPROVED** [review]

**Pull request merged**